index.html

```diff
@@ -1,8 +1,9 @@
 ---
-layout: steps
+layout: default
 ---
 
-<h1>R kernel for Jupyter <img class=logo src="{{site.baseurl}}/images/irkernel-logo.svg"></h1>
+<h1 id="landing-title">R kernel for Jupyter Notebook</h1>
+<img class=logo src="{{site.baseurl}}/images/PoweredbyJupyter-darkborder.svg">
```

Comment:
sure, let’s change it! i’m not too attached to it. but

1. this should be actually mentioned in the guidelines (the PDF): i think you need to specify what goes and what doesn’t.
2. i’d still like to have a logo that uses Jupyter’s visual language.

for the new logo: if not the “shells” then maybe another recognizable part? in any case, i like the fact that R has a halo like saturn… DAMN, we could have named the kernel “satuRn” 😆